tests/cypress/integration/Home/commerceHomePage.cy.js

```diff
@@ -1,56 +1,310 @@
+		cy.get( '.nfd-flex-none > .nfd-button--upsell' ).click();
+		cy.get( '.nfd-notification--success' )
+			.should( 'exist' )
+			.and( 'contain.text', 'Your site is online now!' );
+		cy.contains( 'Ready to go to the next level?', {
+			timeout: customCommandTimeout,
+		} ).should( 'exist' );
+	} );
+} );
+
+describe( 'Commerce Home Page- Live Mode', () => {
+	before( () => {
+		cy.exec( `npx wp-env run cli wp plugin deactivate woocommerce`, {
+			failOnNonZeroExit: false,
+		} );
+		cy.visit( '/wp-admin/admin.php?page=' + pluginId + '#/home' );
+		comingSoon( false );
+	} );
+
+	beforeEach( () => {
+		cy.visit( '/wp-admin/admin.php?page=' + pluginId + '#/home' );
+	} );
+
+	it( 'Verify presense of Ready to go to next level? canvas', () => {
+		cy.get( '.nfd-flex.nfd-gap-4', { timeout: customCommandTimeout } )
+			.eq( 2 )
+			.as( 'readyToGoNextLevel' )
+			.should( 'exist' );
+		cy.get( '@readyToGoNextLevel', { timeout: customCommandTimeout } )
+			.find( 'h1' )
+			.should( 'have.text', 'Ready to go to the next level?' );
+		cy.get( '@readyToGoNextLevel' )
+			.find( 'div span' )
+			.should( 'exist' )
+			.and( 'have.text', 'Your site is live to the world!' );
+	} );
+
+	it( 'Verify by default View Site option should be displayed', () => {
+		cy.contains( '.nfd-button--primary', 'View Site', {
+			timeout: customCommandTimeout,
+		} )
+			.should( 'exist' )
+			.invoke( 'removeAttr', 'target' )
+			.click();
+		cy.url().should( 'eq', Cypress.config().baseUrl + '/' );
+	} );
+} );
+
+describe( 'Commerce Home Page- Next Steps', () => {
+	before( () => {
+		cy.exec( `npx wp-env run cli wp plugin deactivate woocommerce`, {
+			failOnNonZeroExit: false,
+		} );
+	} );
+
+	beforeEach( () => {
+		cy.visit( '/wp-admin/admin.php?page=' + pluginId + '#/home' );
+	});
+	
+	after(() => {
+		cy.exec(`npx wp-env run cli wp option delete nfd_module_onboarding_flow`)
+		cy.reload();
+		cy.visit(
+			'/wp-admin/index.php?page=nfd-onboarding#/wp-setup/step/get-started/experience'
+		);
+		cy.get('button.nfd-nav-card-button.nfd-card-button').should('be.disabled')
+	})
+
+	it( 'Verify Next steps for your site when woocommerce is not active', () => {
+		const steps = [
+			'Add a new page to your site',
+			'Upload media to your site',
+			'Enable Jetpack to connect to your social media accounts',
+		];
+		cy.get( '.nfd-grid.nfd-gap-4', { timeout: customCommandTimeout } )
+			.as( 'nextSteps' )
+			.should( 'exist' );
+		cy.get( '@nextSteps' )
+			.find( 'h1' )
+			.should( 'have.text', 'Next steps for your site' );
+		cy.get( '@nextSteps' )
+			.find( 'p' )
+			.should(
+				'have.text',
+				"You're just a few steps away from sharing your store with the world!"
+			);
+		cy.get( '@nextSteps' )
+			.find( 'ul li' )
+			.each( ( item, index, list ) => {
+				expect( list ).to.have.length( 3 );
+
+				expect( Cypress.$( item ).text() ).to.eq( steps[ index ] );
+			} );
+	} );
+
+	it( 'Verify Next steps when experience level is novice', () => {
+		const steps = [
```

Comment:
@sangeetha-nayak  Can concat `steps` in `other_steps` to avoid duplication of data?

Reply:
Done